src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,39 @@
 import React from 'react';
+import { FullPost } from '../../types/FullPost';
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
-);
+import './PostInfo.scss';
+
+type Props = {
+  post: FullPost;
+};
+
+export const PostInfo: React.FC<Props> = ({ post }) => {
```

Comment:
It's a good practice to use an interface instead of a type for component props. Consider changing `type Props` to `interface Props`.